code_samples/register_actions/resources/META-INF/plugin.xml

```diff
@@ -63,21 +63,45 @@
+      omitted if actions will be added at runtime. See documentation above for attributes.
+    -->
+    <!-- All off the following menu groups add the action SimplePopDialogAction to menus in different ways.
+      Note the action ids are unique. -->
+    <!-- GroupedActions demonstrates declaring an action group using the default ActionGroup implementation provided by the
+      IntelliJ Platform framework. (Note the lack of a "class" attribute.) GroupedActions gets inserted after SimpleAction
+      in the Tools menu. Because the group's implementation is default, it cannot impose enable/disable conditions. Instead it
+      must rely on the conditions imposed by the parent menu where it is inserted. It declares one action in the group. -->
+    <group id="org.jetbrains.tutorials.actions.GroupedActions" text="Example Grouped Actions" popup="true">
+      <add-to-group group-id="ToolsMenu" anchor="after" relative-to-action="org.jetbrains.tutorials.actions.SimpleAction"/>
+      <action class="org.jetbrains.tutorials.actions.SimplePopDialogAction" id="org.jetbrains.tutorials.actions.SimpleGroupedAction"
+              text="A Grouped Action" description="Grouped Action Demo">
       </action>
     </group>
-    <group id="CustomDefaultActionGroup" class="org.jetbrains.tutorials.actions.CustomDefaultActionGroup" popup="true"
-           text="DefaultActionGroup Inheritor" description="Default Action Group Demo">
-      <add-to-group group-id="ToolsMenu" anchor="last"/>
-      <action class="org.jetbrains.tutorials.actions.CustomGroupedAction" id="CustomGroupedAction"
-              text="Custom Grouped Action" description="Custom Grouped Action Demo"/>
+    <!-- CustomDefaultActionGroup demonstrates declaring an action group based on a ActionGroup class supplied by this plugin.
+      This group is to be inserted atop the Editor Popup Menu. It declares one action in the group. -->
+    <group id="org.jetbrains.tutorials.actions.ExampleCustomDefaultActionGroup" class="org.jetbrains.tutorials.actions.CustomDefaultActionGroup" popup="true"
+           text="Example Custom DefaultActionGroup" description="Custom DefaultActionGroup Demo">
+      <add-to-group group-id="EditorPopupMenu" anchor="first"/>
+      <action class="org.jetbrains.tutorials.actions.SimplePopDialogAction" id="org.jetbrains.tutorials.actions.CustomGroupedAction"
+              text="A Custom Grouped Action" description="Custom Grouped Action Demo"/>
     </group>
-    <group id="BaseActionGroup" class="org.jetbrains.tutorials.actions.BaseActionGroup" popup="true"
-           text="ActionGroup Demo" description="Extending AnAction Demo">
-      <add-to-group group-id="ToolsMenu" anchor="first"/>
+    <!-- DynamicActionGroup demonstrates declaring an action group without a static action declaration.
+     An action is added to the group programmatically in the DynamicActionGroup implementation. The group is added
+     at the top of the tools menu. -->
```

Comment:
at the bottom (you're using `anchor="last"`)

Reply:
Careless of me. I changed the menu locations but not the comment.